scripts/fakeprinter.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+
+import sys
+import time
+import serial
+
+fwlist = {
```

Comment:
someother firmware strings that can be added

'aprinter' : """ ok FIRMWARE_NAME:APrinter"""

'teacup' :  """FIRMWARE_NAME:Teacup FIRMWARE_URL:http://github.com/traumflug/Teacup_Firmware/ PROTOCOL_VERSION:1.0 MACHINE_TYPE:Mendel EXTRUDER_COUNT:1 TEMP_SENSOR_COUNT:2 HEATER_COUNT:3"""
(other marlin was a custom version) 

'marlin' : """FIRMWARE_NAME:Marlin 1.1.0-RC7 (Github) SOURCE_CODE_URL:https://github.com/MarlinFirmware/Marlin PROTOCOL_VERSION:1.0 MACHINE_TYPE:3D Printer EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff"""